### PR TITLE
Linter-Setup

### PR DESCRIPTION
**Pull request by belalgamal46**
**The Hello Microverse Project**
   - Created a new repo and pushed it to github
   - Created a new branch and switched to the new branch
   - Created gitignore, index.html, and styles.css files
   - Created the .github file and it's sub folders and linters.yml
   - Created the config files 
   - Added lighthouse library
   - Added Linters for HTML&CSS
   - Added test.md, and node_modules to .gitignore file
   - Created Readme file

### CODE REVIEW

**Review by aimalamiri — COMMENTED** [review]
Hi @belalgamal46, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

Untitled

## Invalid Code Review Request

You have submitted a project, that the pull request has no summary text. Please make sure to add some description to your PR and ask for a second review request.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: README.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello Microverse
+
+Description the project.
```
Comment (victorokoroji) [inline README.md]: - [x] Kindly write a description of your project, this will make your readme professional 👍🏽

File: README.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+Clone repo
+git clone
```
Comment (victorokoroji) [inline README.md]: - [x] Please make sure you write complete information in your readme.  Please write the link used to clone the project 

e.g `git clone https://github.com/belalgamal46/Hello-Microverse.git`

**Review by victorokoroji — CHANGES_REQUESTED** [review]
Hi @belalgamal46 , 

You have done a great job so far! 💪 👏
You still need to work on a few issues before moving on to the next project, but you're almost there!

## Top highlight 🎯

- The Pull Request is following the GitHub Flow ✅ 
- The commit messages are meaningful ✅ 
- Linters are working properly ✅ 


## Required Changes ♻️

- [x]  You should make sure your commit messages are written in the imperative form e.g. instead of writing `Created gitignore file`, you should write` Create gitignore file`

- [x] Kindly ensure that your pull request has a descriptive title telling you what changes you have made so that reviewers and other people using your code will understand what the PR is all about.

Instead of `feature-gitignore`, you can have something like hello-microverse or linter-setup which is more descriptive

_Check the comments under the review._


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Reem-lab — APPROVED** [review]
# Hi @belalgamal46,👋🏻👋🏻

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉🔥🔥💯💯



## Highlights ✨✨
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Great working on the functionality✅
📌 Good commit messages ✅
📌Professional README ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by belalgamal46** [comment]
lkasnl,kasn

**Pull request merged**